### PR TITLE
Upload activity masking layers

### PR DESCRIPTION
**Pull request by zamuzakki**
This is the plugin side for https://github.com/ConservationInternational/cplus-plugin/issues/564

### CODE REVIEW

File: src/cplus_plugin/models/base.py [hunk 1]
```diff
@@ -402,6 +402,7 @@ def from_dict(cls, activity_dict: typing.Dict):
             del pathway["carbon_uuids"]
             pathways.append(NcsPathway(**pathway))
         activity_dict["pathways"] = pathways
+        del activity_dict["mask_uuids"]
```
Comment (Samweli) [inline src/cplus_plugin/models/base.py:405]: Thanks for changes @zamuzakki, one question, where are the `mask_uuids` from?

Comment (zamuzakki) [reply src/cplus_plugin/models/base.py:405]: @Samweli it's coming from the API.

Comment (Samweli) [reply src/cplus_plugin/models/base.py:405]: @zamuzakki what if the key 'mask_uuids' is missing?

Comment (zamuzakki) [reply src/cplus_plugin/models/base.py]: @Samweli It could result in KeyError. I've updated it to `activity_dict.pop("mask_uuids", None)`

**Review by Samweli — APPROVED** [review]

**Pull request merged**